### ISSUE TITLE
Installation fails if `python` is not installed on host system

### ISSUE THREAD

**Issue by Baltoli** [cachix/install-nix-action#140]
On macOS 12.3 Monterey (at least, I've not verified other versions), there is no `python` installed. This causes the action to fail with the following error:
```
> Run cachix/install-nix-action@v17
The node12 is not supported on macOS ARM6[4](https://github.com/runtimeverification/llvm-backend/runs/7196589297?check_suite_focus=true#step:3:5) platform. Use node16 instead.
/Users/user/actions-runner/_work/_actions/cachix/install-nix-action/v17/lib/install-nix.sh: line 41: python: command not found
node:child_process:867
    throw err;
    ^
Error: Command failed: /Users/user/actions-runner/_work/_actions/cachix/install-nix-action/v17/lib/install-nix.sh
(...etc)
```

Would it be possible to add an option to use python3 at [this point](https://github.com/cachix/install-nix-action/blob/451e61183802597c1febd6ca3cf18aa163f93a06/install-nix.sh#L40) in the installation script?

**Comment by domenkozar** [comment]
That's a shame. Is there `python3`?

**Comment by Baltoli** [comment]
Yes, `python3` is available.

**Comment by silviogutierrez** [comment]
It's actually not available on Ubuntu either, at least 20.04 onwards. So self-hosted runners will fail unless they have python installed. See:

https://itsfoss.com/python-not-found-ubuntu/

So the best bet is to document that `python` needs to be available in the runner or maybe try `python3` which should cover most situations.
